packages/core/src/Site/index.js

```diff
@@ -1269,7 +1270,8 @@ class Site {
           })),
       };
 
-      fs.outputJsonAsync(siteDataPath, siteData)
+      // TODO: figure out why the only usage of fse breaks when changed to outputJson.
+      fse.outputJsonAsync(siteDataPath, siteData)
```

Comment:
@ang-zeyu I'm having some trouble at this part; changing this line to `fs.outputJson(siteDataPath, siteData)` causes `test_site/_site/siteData.json` to output the correct json, but incorrectly formatted. For example"
![image](https://user-images.githubusercontent.com/38814428/92993300-0aad2000-f523-11ea-9575-0c6823b14ede.png)

On the other hand, leaving this statement as it is outputs a correctly formatted json file:
![image](https://user-images.githubusercontent.com/38814428/92993313-29131b80-f523-11ea-9549-fd922ded561f.png)

Do you happen to know why this is so? I suspected usage of a JSON.stringify(.., null, 2) but couldn't find any. Or maybe it's some side-effect of another library?

Reply:
I'm assuming the tests fail because `outputJson` is not beautified
Try the options here https://github.com/jprichardson/node-fs-extra/blob/master/docs/outputJson.md

for 3rd party libraries, always go back to the documentation to double check the behaviour first if there are issues =P
For this pr especially, do check if there are obscure differences between the method pairs (e.g. `copy` vs `copyAsync`)

Reply:
noted! I'll be sure to note down the differences, if any